packages/trader/src/Assets/Trading/Categories/contract-type-description-video.tsx

```diff
@@ -34,6 +38,7 @@ const ContractTypeDescriptionVideo = ({ selected_contract_type }: { selected_con
             controls
             width={isMobile() ? 328 : 480}
             height={isMobile() ? 184.5 : 270}
+            data-testid={data_testid}
```

Comment:
this is not the recommended way of doing it.
this is the last option for the selectors. refer to this https://testing-library.com/docs/queries/about/#types-of-queries